src/commands/whoami.rs

```diff
@@ -100,6 +112,8 @@ fn fetch_api_token_email(
         Err(e) => match e {
             ApiFailure::Error(_, api_errors) => {
                 let error = &api_errors.errors[0];
+                // TODO: this message also appears when access token is expired, but it has the user:read scope
```

Comment:
I see. Sorry, I forgot to remove this "TODO". I made a change so that whenever a GlobalUser is created, it will update the OAuth token if it is expired. 

I just tested the case when we use a deleted API token, and it returns an "invalid access token". So, it should be fine?